components/voting.tsx

```diff
@@ -446,24 +447,25 @@ export default class Voting extends React.PureComponent<VotingProps, VotingState
                 />
               </p>
             )}
+
             {this.props.ticketNumberHandling !== TicketNumberWhileVoting.None && (
               <label>
-                Ticket {this.props.ticketsProvider === TicketsProvider.Eventbrite && 'order'} #{' '}
+                Ticket {this.props.ticketsProvider === TicketsProvider.Eventbrite && 'order'} #{' '}{this.props.waitingListCanVoteWithEmail && 'or your waiting list email'}
                 <em>
                   {' '}
                   <span
                     className="fa fa-question-circle"
                     style={{ cursor: 'pointer', fontSize: '20px' }}
                     title={
                       this.props.ticketNumberHandling === TicketNumberWhileVoting.Optional
-                        ? 'Your vote will have a higher weighting if you optionally supply your ticket # from your ticket confirmation email when getting an attendee ticket.'
-                        : 'To submit a vote you must supply your ticket # from your ticket confirmation email when getting an attendee ticket.'
+                        ? VotingWithOptionalTicketMessage
+                        : `To submit a vote you must supply your ${this.props.waitingListCanVoteWithEmail ? 'either' : ''} ticket # from your ticket confirmation email when getting an attendee ticket${this.props.waitingListCanVoteWithEmail ? 'or your email address that you registered with the waiting list.' : ''}.`
```

Comment:
this doesn't read quite right.

`To submit a vote you must supply your either ticket #...`

The either should come before your

`To submit a vote you must supply either your ticket #...`

The second check is missing a space which means the text will be right up next to the previous text, like this `getting an attendee ticketor your email address`